### ISSUE TITLE
Source files without license headers

### ISSUE THREAD

**Issue by puntogil** [kohsuke/tiger-types#2]
Hi
The following source files are without license headers:
./src/test/java/org/jvnet/tiger_types/ListerTest.java
./src/test/java/org/jvnet/tiger_types/TypesTest.java
Please, confirm the licensing of code and/or content/s, and add license headers
https://fedoraproject.org/wiki/Packaging:LicensingGuidelines?rd=Packaging/LicensingGuidelines#License_Clarification

Thanks in advance
Regards

**Comment by madhephaestus** [comment]
Yes it would be helpful for me to also know the license this code is released under :)
